src/core/renderers/webgl/WebGLRenderer.js

```diff
@@ -310,6 +310,7 @@ WebGLRenderer.prototype.setObjectRenderer = function (objectRenderer)
     }
 
     this.currentRenderer.stop();
+    this.currentRenderer.destroy();
```

Comment:
Aha, understood. I'll revisit this in our code base and push an update tomorrow hopefully.